java/src/jmri/configurexml/StoreXmlConfigAction.java

```diff
@@ -75,8 +75,8 @@ public static File getFileCustom(JFileChooser fileChooser) {
         // check for possible overwrite
         if (file.exists()) {
             int selectedValue = JOptionPane.showConfirmDialog(null,
-                    "File " + file.getName() + " already exists, overwrite it?",
-                    "Overwrite file?",
+                    rb.getString("FileOverwriteWarning1") + " " + file.getName() + " " + rb.getString("FileOverwriteWarning2"),
```

Comment:
Right, this was a dirty String trick but Bundle complained about a "String,String" type not supported. Will have to study where the {0} get's filled in. Lots of props using that style, so will convert.